proof-of-us-v1/proof-of-us.pact

```diff
@@ -294,10 +298,38 @@
 
   ;;UTILITY FUNCTIONS
 
+  (defun get-guard-threshold:integer (guards:[guard])
+    (let ((count:integer (length guards)))
+      (if (>= count MIN-MAJORITY-THRESHOLD)
```

Comment:
I think we can suffice here with just MIN-SIGNATURE-THRESHOLD and use

`(if (> count MIN-SIGNATURE-THRESHOLD)`

Reply:
Yes, we can!

I updated the code to reflect that now.